docusaurus/docs/cloud/projects/settings.md

```diff
@@ -15,27 +16,28 @@ import InvoiceStatus from '/docs/snippets/invoices-statuses.md'
 
 # Project settings
 
-From a chosen project's dashboard, the *Settings* tab, located in the header, enables you to manage the configurations and settings for your Strapi Cloud project.
+From a chosen project's dashboard, the ![Settings icon](/img/assets/icons/settings.svg) icon, located in the header, enables you to manage the configurations and settings for your Strapi Cloud project and its environments.
 
-There are 7 tabs available:
+The settings' menu on the left side of the interface is separated into 2 categories: the settings for the entire project and the settings specific to any configured environment for the project.
 
-- ![General icon](/img/assets/icons/Faders.svg) [*General*](#general),
-- ![Backups icon](/img/assets/icons/ArrowClockwise.svg) [*Backups*](#backups),
-- ![Domains icon](/img/assets/icons/Browsers.svg) [*Domains*](#domains),
-- ![Variables icon](/img/assets/icons/code2.svg) [*Variables*](#variables),
+## Project-level settings
+
+There are 5 tabs available for the project's settings:
+- ![General icon](/img/assets/icons/Faders.svg) [*General (project)*](#general),
+- ![Environments icon](/img/assets/icons/v5/Stack.svg) [*Environments*](#environments),
```

Comment:
This link doesn't work, the section is linked as `#environments--` instead of `#environments`

Reply:
FYI `--` means there are badges at the end of the title. But there's indeed a little issue with links to titles with badges so I fixed it manually. Thanks for spotting it though!